### DISCUSSION TITLE
Welcome to img Discussions!

### DISCUSSION THREAD

**Discussion by lainq** [lainq/img#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mjvbz** [comment]
I think you can create a new organization for this and make some cool python librarys for people i think it will be good.

**Comment by lainq** [comment]
Thanks for the suggestion. Maybe I will move upcoming projects to an organization. we'll see 🙂

**Reply by mjvbz** [reply]
M! I will help you with your projects

**Reply by lainq** [reply]
Thanks a lot :)

**Reply by mjvbz** [reply]
😊😊 @pranavbaburaj

**Reply by mjvbz** [reply]
I will surely join you with the community 😊

**Reply by lainq** [reply]
Thanks

**Comment by mjvbz** [comment]
Wwhat about creating an orgainaisation named @theDevshub for making python librarys and website for the community with python library docs. What you think ?

**Reply by lainq** [reply]
Umm. yeah that will be nice

**Reply by mjvbz** [reply]
Mm!

**Reply by mjvbz** [reply]
Why this library doesnt have documentation its better if we add a docs to this repo. May more people start to use this.

**Reply by lainq** [reply]
Thats a great suggestion. i will take a look into it. But, as the library is really small and contains only two methods, the documentation will also be limited.

**Reply by mjvbz** [reply]
hmm! You are right!! Any ideas for a new library with python not small something big which u think we can earn some stars and some good ampunt of users. I will help you with the library.

**Reply by mjvbz** [reply]
Any lib ideas ?

**Reply by lainq** [reply]
No, nothing yet.

**Comment by mjvbz** [comment]
can i create a doc for it as my repository not as a doc for the repo. i mean like uses of this and where is the source code is in an .md file in my one of github repo. can i ?

**Comment by lainq** [comment]
Sure
